src/tokens/AssetBalanceFiat.tsx

```diff
@@ -0,0 +1,68 @@
+import BigNumber from 'bignumber.js'
```

Comment:
this component looks similar to the one on the home screen but it will eventually have this [tool tip thing](https://www.figma.com/file/LZJAPf1CdLSRGujzQHCbXj/In-app-dapp-actions?type=design&node-id=225-2383&t=1xosKbu8Dhi99Q5P-0). that's why this is a new component

Reply:
I think we should avoid differences and use the same component. Otherwise we risk having inconsistencies.

Reply:
thanks for this! i actually misunderstood what TokenBalance does, I thought that it was the whole component (with label and buttons etc) but it's not 😅 so i could reuse it